src/components/Envelope.vue

```diff
@@ -1047,10 +1047,6 @@ export default {
 	display: flex;
 	width: 200px;
 }
-:deep(.list-item--one-line) {
-	padding: 10px;
-	margin: 2px;
-}
```

Comment:
Oops, that was fix for the clickable area. I'll push it again.

Reply:
Hmm, I should learn to read :see_no_evil: The changes are already upstreamed. I'll revert again.